clowdapp.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: floorist
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: floorist
+  spec:
+    database:
+      sharedDbAppName: ${SHARED_DB_APP_NAME}
+    objectStore:
+      - ${BUCKET_NAME}
+    envName: ${ENV_NAME}
+    jobs:
+    - name: metrics-exporter
+      schedule: '*/5 * * * *'
+      podSpec:
+        image: ${IMAGE}:${IMAGE_TAG}
+        env:
+        # TODO: Grab these from object store and remove from template
+        - name: AWS_ACCESS_KEY_ID
+          value: "from a secret"
+        - name: AWS_SECRET_ACCESS_KEY
+          value: "from a secret"
+        - name: AWS_REGION
+          value: "from a secret"
+        - name: AWS_BUCKET
+          value: "from a secret"
+        - name: AWS_ENDPOINT
+          value: "from a secret"
+            
+        - name: FLOORPLAN_FILE
+          value: "from a secret"
```

Comment:
No need to be a secret, or maybe not even a dynamic parameter, we can just make this something hardcoded and mount the configmap accordingly.

Reply:
Yeah - this was an early WIP - I changed this to read from a ConfigMap